### ISSUE TITLE
Devportal API Documentation page accessibility issues

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#8383]
### Describe your problem(s)

1. Doc title and `docs` Text does not meet the minimum AA contrast ratio (4.5:1).

![image](https://user-images.githubusercontent.com/20179540/84979659-9a724680-b14d-11ea-8ef0-e0a734729247.png)

2. hide button has no keyboard focus

### How will you implement it

---
